pkg/limayaml/validate.go

```diff
@@ -136,6 +136,12 @@ func Validate(y *LimaYAML, warn bool) error {
 		return fmt.Errorf("field `memory` has an invalid value: %w", err)
 	}
 
+	for i, disk := range y.AdditionalDisks {
+		if disk.Name == "" {
```

Comment:
This is not strict enough. `store.DiskDir` requires the name to be a valid identifier: https://github.com/lima-vm/lima/blob/2011af722e22d3c3663d0425768b38d760433b4d/pkg/store/store.go#L104-L105

For example:

```console
❯ l disk create "foo bar" --size 1gb
FATA[0000] identifier "foo bar" must match ^[A-Za-z0-9]+(?:[._-](?:[A-Za-z0-9]+))*$: invalid argument
```

So you could maybe just call `DiskDir` for validation:

```go
if _, err := store.DiskDir(disk.Name); err != nil {
    return fmt.Errorf("field `additionalDisks[%d].name is invalid`: %w, err)
}
```

Reply:
Turns out you cannot import `store` into `limayaml` because that creates a circular import. This will change once the validation moves to `limatmpl`, but we are not there yet, so I guess you need to call `identifiers.Validate(disk.Name)` directly.